src/api/authentication.js

```diff
@@ -217,5 +225,42 @@ export async function authenticate (ctx, next) {
   return next()
 }
 
+export async function getEnabledAuthenticationTypes (ctx, next) {
+  // Log level for messages logged
+  let logLevel = 'error'
+
+  try {
```

Comment:
@bradsawadye Why did you surround everything in a `try-catch`?

Reply:
addressed in https://github.com/jembi/openhim-core-js/pull/1082/commits/75107c24ed1e86da1944b25de6bf4ff4368c007a